### PR TITLE
Playwright: deflake verification tests

### PR DESCRIPTION
**Pull request by richvdh**
Wait for the QR code to be rendered, in an attempt to deflake these two

Fixes: https://github.com/element-hq/element-web/issues/27819
Fixes: https://github.com/element-hq/element-web/issues/27789

### CODE REVIEW

**Review by t3chguy — APPROVED** [review]

File: playwright/e2e/crypto/user-verification.spec.ts [hunk 1]
```diff
@@ -101,20 +106,29 @@ test.describe("User verification", () => {
...
         // request verification by emoji
         await page.locator("#mx_RightPanel").getByRole("button", { name: "Verify by emoji" }).click();
 
         /* on the bot side, wait for the verifier to exist ... */
         const botVerifier = await awaitVerifier(bobVerificationRequest);
-        // ... confirm ...
-        botVerifier.evaluate((verifier) => verifier.verify()).catch(() => {});
+        // ... and confirm. We expect the verification to fail; we catch the error on the DOM side
+        // to stop playwright marking the evaluate as failing in the UI.
+        const botVerification = botVerifier.evaluate((verifier) => verifier.verify().catch(() => {}));
```
Comment (richvdh) [inline playwright/e2e/crypto/user-verification.spec.ts:121]: this is a driveby cleanup, not really related to the deflaking.
